### PR TITLE
RA2 Ch2,4 vDU requirements

### PR DESCRIPTION
**Pull request by petorre**
partly closes [#3447](https://github.com/anuket-project/anuket-specifications/issues/3447)

### CODE REVIEW

**Review by TFredberg — APPROVED** [review]

**Comment by rgstori** [comment]
we should reference a "real-time" profile extension from RM 2.4.2

**Comment by petorre** [comment]
> we should reference a "real-time" profile extension from RM 2.4.2

It is now in RM as "rt-tsn" [here](https://github.com/anuket-project/RM/blob/7ed7816eb8037f7a01cd7583ed5213c19bc9c191/doc/ref_model/chapters/chapter02.rst?plain=1#L522).

Where and how should we reference it in RA2?

File: doc/ref_arch/kubernetes/chapters/chapter02.rst [hunk 1]
```diff
@@ -145,6 +145,18 @@ Cloud Infrastructure Software Profile Capabilities
      - n/a (2)
      - *n/a (2)*
      - N/A
+   * - Exposed Infrastructure Capabilities
+     - e.cap.026
+     - Real-Time settings
+     - No
+     - Optional
```
Comment (rgstori) [inline doc/ref_arch/kubernetes/chapters/chapter02.rst:152]:
```suggestion
     - Mandatory for flavour `rt-tsn`
```

File: doc/ref_arch/kubernetes/chapters/chapter02.rst [hunk 2]
```diff
@@ -145,6 +145,18 @@ Cloud Infrastructure Software Profile Capabilities
...
+     - 
+   * - Exposed Infrastructure Capabilities
+     - e.cap.027
+     - Time Sensitive Networking with PTP Hardware Clock and syncronization with SyncE
+     - No
+     - Optional
```
Comment (rgstori) [inline doc/ref_arch/kubernetes/chapters/chapter02.rst:158]:
```suggestion
     - Mandatory for flavour `rt-tsn`
```

File: doc/ref_arch/kubernetes/chapters/chapter02.rst [hunk 3]
```diff
@@ -145,6 +145,18 @@ Cloud Infrastructure Software Profile Capabilities
      - n/a (2)
      - *n/a (2)*
      - N/A
+   * - Exposed Infrastructure Capabilities
+     - e.cap.026
+     - Real-Time settings
+     - No
+     - Mandatory for flavour `rt-tsn`
+     -
```
Comment (rgstori) [inline doc/ref_arch/kubernetes/chapters/chapter02.rst:153]:
```suggestion
     -
```

**Pull request merged**